src/libcore/default.rs

```diff
@@ -115,6 +115,48 @@ pub trait Default: Sized {
     fn default() -> Self;
 }
 
+/// Equivalent to [`Default::default()`] but without repeating the word
+/// "default" twice at the call site.
+///
+/// For example:
+/// ```
+/// #![feature(default_free_fn)]
+///
+/// use std::default::default;
+///
+/// #[derive(Default)]
+/// struct AppConfig {
+///     foo: FooConfig,
+///     bar: BarConfig,
+/// }
+///
+/// #[derive(Default)]
+/// struct FooConfig {
+///     foo: i32,
+/// }
+///
+/// #[derive(Default)]
+/// struct BarConfig {
+///     bar: f32,
+///     baz: u8,
+/// }
+///
+/// fn main() {
+///     let options = AppConfig {
+///         foo: default(),
+///         bar: BarConfig {
+///             bar: 10,
+///             ..default(),
+///         },
+///     };
+/// }
+/// ```
+#[unstable(feature = "default_free_fn", issue = "none")]
+#[inline]
+pub fn default<Value: Default>() -> Value {
```

Comment:
I think we should use `T` instead of `Value` for the name of this type parameter. Both mean "pretty much any type" but `T` is widely used in the standard library for that purpose.

Reply:
Done.